Modules/vis/visualisations/common/vis.helper.js

```diff
@@ -41,38 +41,7 @@ var view =
   {
     this.start = ((new Date()).getTime())-(3600000*24*time);	//Get start time
     this.end = (new Date()).getTime();	//Get end time
-  },
-  
-  'round_interval':function(interval)
```

Comment:
Here's the forum thread where I added this to fix an issue with the multigraph: https://openenergymonitor.org/emon/node/12490, dont mind too much about the use of ifs in this case - although perhaps it could rewritten by iterating over an array of allowed intervals..